lib/utils.pm

```diff
@@ -1947,6 +1947,10 @@ sub install_patterns {
         if (($pt =~ /sap_server/) && is_sle('=11-SP4')) {
             next;
         }
+        # skip the installation of "fips" for SLED cases, poo#98745.
+        if (($pt =~ /fips/) && check_var('SLE_PRODUCT', 'sled')) {
```

Comment:
Need we limit this condition for special sle version such as '15-SP4+'?

Reply:
No, the limit condition is that the product is sled. It has nothing to do with product version.